test/generators/scaffold_controller_generator_test.rb

```diff
@@ -1,4 +1,5 @@
 require 'test_helper'
+require 'generators/rails/resource_override'
```

Comment:
I'm kind of surprised this is required.  Have you tested that it works outside of test on a rails app?